lib/tutorials/index.js

```diff
@@ -68,10 +68,48 @@ const getTags = async (done) => {
     if (!response.ok) {
         throw new Error(`failed to download tags (${response.status})`);
     }
+    var tutorialCategories = new Set([
+        "animation",
+        "assets",
+        "audio",
+        "basics",
+        "camera",
+        "collision",
+        "culling",
+        "events",
+        "facebook",
+        "games",
+        "html",
+        "input",
+        "lighting",
+        "loading",
+        "materials",
+        "mobile",
+        "mouse",
+        "multiplayer",
+        "networking",
+        "particles",
+        "pbr",
+        "physics",
+        "posteffects",
+        "procedural",
+        "raycast",
+        "rendering",
+        "resolution",
+        "scenes",
+        "scripts",
+        "shaders",
+        "textures",
+        "time",
+        "touch",
+        "ui",
+        "video",
+        "vr"
+    ]);
```

Comment:
Wouldn't it be better to just ensure starter kits have a `starterkit` tag and just discard anything with `starterkit` when building a list of tutorials? Otherwise, we're losing the data-driven nature of this.

Reply:
That could work, but it would mean that we would have a new similar condition excluding every new tag (apart from `starterkit`) we create through the Admin panel, which is why I thought maybe this approach was more scalable?

Reply:
> Wouldn't it be better to just ensure starter kits have a starterkit tag and just discard anything with starterkit when building a list of tutorials? Otherwise, we're losing the data-driven nature of this.

This generates the list the on left hand side here 
<img width="742" alt="image" src="https://user-images.githubusercontent.com/16639049/173598571-15bd5895-8d73-46fc-8ab4-272eeb9be65a.png">

Going along the line of thought here, we would have to get all the projects with 'tutorial' tag and create a list of tags like 'mobile' and 'vr' from that to generated the menu on the left.

Reply:
@josflesan I'm not sure what you mean. Searching for all projects that include a `starterkit` tag to get all starter kits or all projects that have `tutorial` to get all tutorials seems very scalable.

@yaustar Yep.